scripts/carrot/carrotCli.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+#carrot_cli software create --name example-test --description "An example CARROT test software" --repository_url https://github.com/broadinstitute/carrot-example-test
```

Comment:
This line can probably just be removed since it's not being used and is maybe a bit confusing removed from the context of the example test repo.